app/src/main/res/values/styles.xml

```diff
@@ -1,10 +1,14 @@
 <resources>
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
         <!-- Customize your theme here. -->
         <item name="colorPrimary">@color/colorPrimary</item>
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <style name="FloatingHintStyle" parent="@style/TextAppearance.AppCompat">
+        <item name="android:textColor">@color/black</item>
```

Comment:
Change color from black to this 
![image](https://user-images.githubusercontent.com/3202599/142173730-7e873229-44cd-4774-83c4-148fdcd25bcb.png)

Reply:
okay will do that